src/object.d

```diff
@@ -2065,8 +2065,8 @@ class TypeInfo_Const : TypeInfo
     }
 
     override size_t getHash(scope const void *p) const { return base.getHash(p); }
-    override bool equals(in void *p1, in void *p2) const { return base.equals(p1, p2); }
-    override int compare(in void *p1, in void *p2) const { return base.compare(p1, p2); }
+    override bool equals(const scope void *p1, const scope void *p2) const { return base.equals(p1, p2); }
+    override int compare(const scope void *p1, const scope void *p2) const { return base.compare(p1, p2); }
```

Comment:
`base` can be the `TypeInfo` for a `class` or `struct` which implements `opEquals`|`opCompare` without using the `scope` attribute.

Reply:
What do you recommend?